content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md

```diff
@@ -20,6 +20,46 @@ You can configure this admission controller to set cluster-wide defaults and [ex
 ## Configure the Admission Controller
 
 {{< tabs name="PodSecurityConfiguration_example_1" >}}
+{{% tab name="pod-security.admission.config.k8s.io/v1" %}}
+```yaml
+apiVersion: apiserver.config.k8s.io/v1
+kind: AdmissionConfiguration
+plugins:
+- name: PodSecurity
+  configuration:
+    apiVersion: pod-security.admission.config.k8s.io/v1
+    kind: PodSecurityConfiguration
+    # Defaults applied when a mode label is not set.
+    #
+    # Level label values must be one of:
+    # - "privileged" (default)
+    # - "baseline"
+    # - "restricted"
+    #
+    # Version label values must be one of:
+    # - "latest" (default) 
+    # - specific version like "v{{< skew currentVersion >}}"
+    defaults:
+      enforce: "privileged"
+      enforce-version: "latest"
+      audit: "privileged"
+      audit-version: "latest"
+      warn: "privileged"
+      warn-version: "latest"
+    exemptions:
+      # Array of authenticated usernames to exempt.
+      usernames: []
+      # Array of runtime class names to exempt.
+      runtimeClasses: []
+      # Array of namespaces to exempt.
+      namespaces: []
+```
+
+{{< note >}}
+`pod-security.admission.config.k8s.io/v1` configuration requires v1.25+. For v1.23 and v1.24, use v1beta1.
```

Comment:
We could mention v1.22 here.

It's also feasible to link to this section of the same page but in the v1.22 (or whatever) docs. That feels like a nice-to-have for a follow up PR, if we even want it.

Reply:
I like the link idea... that actually lets us drop the tabs, which got unwieldy with multiple versions and such a long apiVersion